public/scripts/char-data.js

```diff
@@ -70,6 +70,8 @@
  * @property {"system" | "user" | "assistant"} depth_prompt.role - The role the character takes on during the prompted interaction (system, user, or assistant).
  * // Non-standard extensions added by external tools
  * @property {string} [pygmalion_id] - The unique identifier assigned to the character by the Pygmalion.chat.
+ * @property {`${string}/${string}`} [github_repo] - The gitHub repository associated with the character.
```

Comment:
I'm not sure that's a valid type. Can you just leave it as a string? In the end, it resolves to `string/string` being the only allowed value.

Reply:
fixed in [`cd95028` (#2255)](https://github.com/SillyTavern/SillyTavern/pull/2255/commits/cd950287a3617363985d08152a0a8d668f0f1db5)